client/app/core/translations/Messages_fr_FR.xml

```diff
@@ -199,6 +199,9 @@
    <translation id="global_means_refunds" qtlid="139923">Mes avoirs</translation>
    <translation id="global_means_fidelity" qtlid="260606">Mon espace fidélité</translation>
    <translation id="global_means_credits" qtlid="448907">Mes crédits</translation>
+   <translation id="global_admin_info">{0} <span class="small">Administrateur</span></translation>
```

Comment:
There is no benefit to provide that kind of sentences to the translation process.

`Administrateur|Technique|Facturation` is enough.

Prefer using ngBind and concatenation if needed.

Reply:
Thanks for the feedback @antleblanc 
We now have only `Administrateur|Technique|Facturation` in the translations
Since a portion of the text has to appear in smaller font, I guess concatenation is not an option. Please feel to suggest any better solution :)